modules/localstack/localstack_legacy_mode_test.go

```diff
@@ -0,0 +1,29 @@
+package localstack
```

Comment:
should be removed?

Reply:
Nope, because we want to check that it will raise the error